strawberryfields/engine.py

```diff
@@ -723,3 +732,20 @@ class Engine(LocalEngine):
 
     # alias for backwards compatibility
     __doc__ = LocalEngine.__doc__
+
+
+class BosonicEngine(LocalEngine):
```

Comment:
This engine doesn't appear in the docs. Would that be intentional? (i.e., we wouldn't like users to get to know that a custom engine is being used)

Reply:
That is not intentional. Can you possibly help figure out why it's not appearing in the docs?

Reply:
Sure! `"BosonicEngine"` will need to be added to `__all__` on line 36. Some engines, like `RemoteEngine` are not added there, because they are available simply from the top level `__init__.py` (like `RemoteEngine`).